src/templates/static.jsx

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable react/prop-types */
+import { graphql } from 'gatsby';
+import React from 'react';
+
+import Content from 'components/shared/content';
+import Layout from 'components/shared/layout';
+import SEO_DATA from 'constants/seo-data';
+
+const StaticPage = ({
+  data: {
+    mdx: { body, frontmatter },
+  },
+}) => (
+  <Layout seo={SEO_DATA.static({ title: frontmatter.title })} headerTheme="white">
+    <article>
+      <div className="safe-paddings py-48 3xl:py-44 2xl:py-40 xl:py-32 lg:py-12 md:py-6">
```

Comment:
Why do we have this unnecessary wrapper?

Reply:
Also, please think about why bottom padding is wrong and what values we should use